### DISCUSSION TITLE
Tengo un problema para instalarlo en WSL

### DISCUSSION THREAD

**Discussion by josedejesuschavez** [CodelyTV/dotly#78] [Q&A]
Ejecuto el comando de instalación y aparece lo siguiente, previamente instale Rust

![image](https://user-images.githubusercontent.com/2577473/105538534-2686b680-5cb9-11eb-8db6-4c35867e9dd1.png)

**Comment by gtrabanco** [comment]
Instala el paquete `cmake` a ver si funciona...

**Comment by andreuSignes** [comment]
Yo lo solucione instalando este paquete
`sudo apt install build-essential`

Lo saque de esta [web](https://ostechnix.com/how-to-fix-rust-error-linker-cc-not-found-on-linux/)
